recipes/role-based-access/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+apiUrl=cube:4000/cubejs-api/v1/load
+
+operatorToken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoib3BlcmF0b3IiLCJpYXQiOjE2Mjg3NDUwNDUsImV4cCI6MTgwMTU0NTA0NX0.VErb2t7Bc43ryRwaOiEgXuU5KiolCT-69eI_i2pRq4o
+managerToken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoibWFuYWdlciIsImlhdCI6MTYyODc0NTAxMSwiZXhwIjoxODAxNTQ1MDExfQ.1cOAjRHhrFKD7Tg3g57ppVm5nX4eI0zSk8JMbinfzTk
+
+query=$(cat curl/query.json)
+
+# wait for the Cube API ready
+sleep 15
```

Comment:
You have a dependency on the Cube container. Isn't it enough? If it's not, can we wait for `/readyz`/`livez` instead of waiting for 15 seconds? https://cube.dev/docs/rest-api#api-reference-readyz

Reply:
That's a good point! I added `/readyz` check to the script.